modules/developer_manual/examples/core/scripts/curl/dav/public_files/view_public_link.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+##
+## Variable Declaration
+##
+USERNAME={oc-examples-username}
+PASSWORD={oc-examples-password}
+API_PATH="/remote.php/dav/public-files/<share_token>"
+SERVER_URI="{oc-examples-server-url}"
+REQUEST_BODY=$(cat <<EOF                                                                
+<?xml version="1.0"?>                                 
+<d:propfind xmlns:d="DAV:" xmlns:oc="http://owncloud.org/ns">
+    <d:prop>   
+        <oc:public-link-item-type />
+        <oc:public-link-item-permission />
+        <oc:public-link-item-expiration />
+        <oc:public-link-item-share-datetime />
+        <oc:public-link-item-owner />
+    </d:prop>
+</d:propfind>
+EOF
+)
+
+curl "'$SERVER_URI/$API_PATH" \
+    -H 'Content-Type: application/xml; charset=UTF-8' \
+    -H 'Depth: 1' \
+    -X PROPFIND \
+    --data-binary "${REQUEST_BODY}" \
+    --user "${USERNAME}:${PASSWORD}" | xml_pp
```

Comment:
this is just wrong - we are talking about public link access - there is no user and password.

in case the public link is protected with a password: use 'public' as user name and the share link password as password